docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -520,9 +520,39 @@ spec:
 apiVersion: v1
 kind: query
 spec:
-  name: Get running docker containers
+  name: Get applications hogging memory
+  platforms: macOS, Linux, Windows
+  description: Returns top 10 applications or processes hogging memory the most.
+  query: SELECT pid, name, ROUND((total_size * '10e-7'), 2) AS memory_used FROM processes ORDER BY total_size DESC LIMIT 10;
+  purpose: Informational
+  contributors: DominusKelvin
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get Mac and Linux machines with unencrypted primary disks
   platforms: macOS, Linux
-  description: Returns the running Docker containers.
-  query: SELECT id, name, image, image_id, state, status FROM docker_containers WHERE state = "running";
+  description:
+  query: SELECT * FROM mounts m, disk_encryption d WHERE m.path= "/" AND m.device = d.name AND d.encrypted = 0;
+  purpose: Informational
+  contributors: DominusKelvin
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get applications hogging memory
```

Comment:
Ah, it looks like there are two "Get applications hogging memory" queries. Is this intentional? These duplicate queries are causing the "Deploy Fleet Website" test to fail because fleetdm.com/queries/:slug using the query's name for the slug.